src/app/cart/cart.component.js

```diff
@@ -9,6 +10,10 @@ import designationsService from 'common/services/api/designations.service';
 import template from './cart.tpl';
 import templateModal from 'app/productConfig/productConfigModal.tpl';
 
+//local dev
+import localDevNav from 'common/localDev/nav/local-dev-nav.component';
```

Comment:
The cart component shouldn't depend on dev stuff right? Otherwise it'll end up in production code. The main module pulled those components together before. We could use a main component that supports transclusion to easily choose what components go inside.